### PR TITLE
enhancement: remove console warnings

### PR DESCRIPTION
**Pull request by MarkNerdi996**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/atoms/badges/NetworkIconBadge.svelte [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
     import { FontWeight, NetworkIcon, Text, Tooltip } from 'shared/components'
     import { Position } from 'shared/components/enums'
 
-    export let network: IPersistedNetwork
-    export let chainId: string
-    export let width: number
-    export let height: number
+    export let networkId: NetworkId
+    export let chainId: number | undefined = undefined
+    export let tooltipText: string = ''
```
Comment (Tuditi) [inline packages/shared/components/atoms/badges/NetworkIconBadge.svelte:8]: The text is not visible anymore for assets.

File: packages/shared/components/molecules/SelectorInput.svelte [hunk 2]
```diff
@@ -108,7 +108,7 @@
                             </Text>
                         </div>
                         {#if option.value && option.key}
-                            <Text type={TextType.pre} fontSize="sm" color="gray-600" classes="flex-1">
+                            <Text type={TextType.pre} fontSize="sm" color="gray-600" classes="flex-1 text-end">
```
Comment (Tuditi) [inline packages/shared/components/molecules/SelectorInput.svelte:111]: What is the reason for adding text-end here?

**Review by Tuditi — CHANGES_REQUESTED** [review]
One change requested. The following warning is thrown from Time.svelte:

```
A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
```

That's the only warning I see that is left, so great job! :clap:

File: packages/shared/components/molecules/SelectorInput.svelte (hunk 2 above)
Comment (MarkNerdi996) [reply packages/shared/components/molecules/SelectorInput.svelte:111]: Without `text-end`
image

With `text-end`
image

File: packages/shared/components/molecules/AssetIcon.svelte [hunk 3]
```diff
@@ -86,7 +86,13 @@
     </div>
     <span class="absolute flex justify-center items-center bottom-0 right-0">
         {#if asset.verification.verified === true}
-            <NetworkIconBadge width={10} height={10} networkId={$activeProfile.network.id} {chainId} />
+            <NetworkIconBadge
+                width={10}
+                height={10}
+                networkId={$activeProfile.network.id}
+                {chainId}
+                tooltipText={$activeProfile?.network?.name}
```
Comment (Tuditi) [inline packages/shared/components/molecules/AssetIcon.svelte:94]: This should be changed to take into consideration the chain name if applicable.

image

**Review by Tuditi — CHANGES_REQUESTED** [review]

**Review by Tuditi — APPROVED** [review]
LGTM! :fire:

**Pull request merged**